includes/class-piklist-add-on.php

```diff
@@ -218,17 +218,19 @@ public static function is_active($add_on = '')
   public static function current()
   {
     $backtrace = debug_backtrace();
-    
     foreach ($backtrace as $trace)
     {
-      if (strstr($trace['file'], '/parts/'))
+	  $file = isset($trace['file']) ? $trace['file'] : '';
```

Comment:
I did this because I was getting a warning that $trace does not have a 'file' item. Actually I've updated this part, to continue the foreach loop instead of unnecessary checking an empty string.